routes/web/hardware.php

```diff
@@ -178,13 +179,17 @@ function () {
         Route::post('bulkcheckout',
             [BulkAssetsController::class, 'storeCheckout']
         )->name('hardware.bulkcheckout.store');
+
     });
 
 Route::resource('hardware', 
         AssetsController::class, 
         [
             'middleware' => ['auth'],
-            'parameters' => ['asset' => 'asset_id'
+            'parameters' => ['asset' => 'asset_id',
+                'names' => [
+                'show' => 'view',
```

Comment:
I feel like the nesting is messed up here somehow - do we have too few `]` or is it just the indenting that's weird?

Reply:
Agree on this one but I need to dig a little deeper

Reply:
It looks weird but we're not actually missing anything. Here it is formatted:
```php
Route::resource(
    'hardware', 
    AssetsController::class, 
    [
        'middleware' => ['auth'],
        'parameters' => [
            'asset' => 'asset_id',
            'names' => ['show' => 'view'],
        ],
    ]
);
```